lib/prometheus/alert_client/client.rb

```diff
@@ -0,0 +1,116 @@
+# encoding: UTF-8
+
+require 'json'
+require 'faraday'
```

Comment:
Really up to you but take a look at the discussion in
https://github.com/abonas/kubeclient/issues/237
and consider
https://github.com/nahi/httpclient 

faraday should be okay though, and is extensively used in ManageIQ.